<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Time per question</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="groupEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Eg:- Koshi,Karnali</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Group Names</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(separate with commas )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Question file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="done_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>440</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="folderButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Media folder</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
